omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ConceptResource1_8.java

```diff
@@ -462,9 +462,14 @@ public static List<ConceptMap> getMappings(Concept instance) {
 	 * @param instance the delegate instance to get the display name off
 	 */
 	@PropertyGetter("display")
-	public String getDisplayName(Concept instance) {
-		ConceptName cn = instance.getName();
-		return cn == null ? null : cn.getName();
+	public String getDisplayString(Concept instance) {
```

Comment:
Not 100% clear to me why you are changing the method name here?

Reply:
I can probably get rid of that, I think originally I was trying to make it consistent with a method I was overriding, but that doesn't look to be the case.

Reply:
Yeah, getDisplayString is the consistent name for the method annotated with "display" outside of this class, so just renaming for consistency and to make sure any overrides need to be deliberately set.